proto/beacon/p2p/v1/types.proto

```diff
@@ -116,4 +163,18 @@ message DepositMessage {
 
   // Deposit amount in gwei.
   uint64 amount = 3;
-}
\ No newline at end of file
+}
+
+// SyncCommittee serves as committees to facilitate light client syncing to beacon chain.
+message SyncCommittee {
+  repeated bytes pubkeys = 1  [(ethereum.eth.ext.ssz_size) = "512,48"];
+  bytes aggregate_pubkey = 2 [(ethereum.eth.ext.ssz_size) = "48"];
+}
+
+// SyncCommitteeSigningData to be used for hash tree root then sign by sync committee participants.
```

Comment:
@terencechain can this be made more clear?

Reply:
`to be used to sign over then check whether the aggregator is selected within the subcommittee`

Also, the latest release changed the name. Do you mind update it?
```python
class SyncAggregatorSelectionData(Container):
    slot: Slot
    subcommittee_index: uint64
```
https://github.com/ethereum/eth2.0-specs/blob/dev/specs/altair/validator.md#syncaggregatorselectiondata

Reply:
Both of these sound good